src/sentry/sentry_metrics/querying/metadata/metrics_code_locations.py

```diff
@@ -72,11 +72,15 @@ class CodeLocationsFetcher:
     # The maximum number of keys that can be fetched by the fetcher.
     #
     # The estimation was naively done by supposing at most 10 metrics with 2 projects and at most 90 timestamps.
```

Comment:
This does not match the `MAXIMUM_KEYS` anymore.

I assume something changed so this can be actually less now?

Reply:
Yep, thanks for noticing that!